Shared/Samples/Add raster from service/AddRasterFromServiceView.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  AddRasterFromServiceView.swift
+//  Samples
+//
+//  Created by Christopher Webb on 6/3/24.
+//  Copyright © 2024 Esri. All rights reserved.
+//
+
+import SwiftUI
+import ArcGIS
+
+struct AddRasterFromServiceView: View {
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// The current viewpoint of the map view.
+    @State private var viewpoint: Viewpoint?
+    
+    /// A Boolean value indicating whether a download operation is in progress.
+    @State private var isDownloading = false
```

Comment:
Probably call this `isLoading` as [`Loadable`](https://developers.arcgis.com/swift/programming-patterns/loading-resources-asynchronously/) is the term for the concept. Also change the prompt texts to "Loading…"